virtcontainers/sandbox.go

```diff
@@ -1938,9 +1942,23 @@ func (s *Sandbox) calculateSandboxCPUs() uint32 {
 				mCPU += utils.CalculateMilliCPUs(*cpu.Quota, *cpu.Period)
 			}
 
+			//todo we should handle error
+			set, err := cpuset.Parse(cpu.Cpus)
```

Comment:
should you just have some simple sanity checks such as `cpu.Cpus != ""` then handle the error as you know it's invalid configuration?

Reply:
Ah, I think I should update the comment.  The simple sanity check is handled within Parse itself, and if we do receive an invalid string (ie, something where Atoi fails), I think we should error out/stop. 

My initial thought is I should just remove then comment/1945.  WDYT @crosbymichael

Reply:
ok, sounds good